scripts/pylint_custom_plugin/regex_path_filter.py

```diff
@@ -0,0 +1,22 @@
+import re
+from pylint.utils import utils
+
+class RegexPathFilter:
+    def __init__(self):
+        self.old_expand_modules = utils.expand_modules
+        utils.expand_modules = self.new_expand_modules
+    
+    def new_expand_modules(self, files_or_modules, black_list, black_list_re, **kwargs):
+        modules, errors = self.old_expand_modules(files_or_modules, black_list, black_list_re, **kwargs)
+        new_modules = self.filtered(modules)
+        return new_modules, errors
+
+    def filtered(self, modules):
+        paths = [r"sdk\\storage\\.*\\_shared\\models.py", r"sdk\\storage\\.*\\_models.py", r"sdk\\storage\\.*\\.*_lease.*", r"sdk\\storage\\.*\\.*client.*"]
```

Comment:
does this work on non-windows machines?

Reply:
Supports posix formatting too, I can add both in and see!